pleskdistup/common/src/util.py

```diff
@@ -9,31 +9,63 @@
 # Returns standard output
 def logged_check_call(cmd: typing.Union[typing.Sequence[str], str], **kwargs) -> str:
     log.info(f"Running: {cmd!r}. Output:")
```

Comment:
So basically if the logging level is above INFO, then the log will just contain some stderr strings w/o actually referencing the command that was executed.

Reply:
Prepended cmd[0] to the logging lines.

Reply:
`f"{cmd[0]}: {line}"`
Not sure about process pid, worth to add it?

Reply:
PID is transient, and there's no real concurrent execution, so there's not much use in it.

My point was more about inconsistency of having stdout and stderr logged with different levels. So alternative solution would be to log everything with INFO, but with prefix like `stdout: ` and `stderr: `.

Alternatively, you could leave this as is, but the current solution:
1. adds extra ERROR lines to the log, which are not necessarily errors;
2. adds extra ERROR words on the console, which might spook users.